From: Boyuan Yang <byang@debian.org>
Date: Wed, 7 Jul 2021 17:10:25 -0400
Subject: Use system cJSON library

---
 tests/testcase.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/testcase.h b/tests/testcase.h
index 5d8f543..3e834f3 100644
--- a/tests/testcase.h
+++ b/tests/testcase.h
@@ -6,7 +6,7 @@
 
 #include "avif/avif.h"
 
-#include "cJSON.h"
+#include <cjson/cJSON.h>
 
 #ifdef __cplusplus
 extern "C" {
